libraries/mediaplayer/impl/src/main/kotlin/io/element/android/libraries/mediaplayer/impl/SimplePlayer.kt

```diff
@@ -67,6 +70,7 @@ class SimplePlayerImpl(
         p.addListener(object : Player.Listener {
             override fun onIsPlayingChanged(isPlaying: Boolean) = listener.onIsPlayingChanged(isPlaying)
             override fun onMediaItemTransition(mediaItem: MediaItem?, reason: Int) = listener.onMediaItemTransition(mediaItem)
+            override fun onTimelineChanged(timeline: Timeline, reason: Int) = listener.onTimelineChanged(timeline)
```

Comment:
The reason for adding this is to allow seeking the track before it is played which requires the track duration.

At the point `onMediaItemTransition` is called,  the track duration is not available (the value is an 'unset' error code). Previously we only recieved the track duration after playing the track. I added `onTimelineChanged` because it seemed like the most relevant callback and, at the point it's called, the duration is available.